update_twiki.py

```diff
@@ -3,48 +3,104 @@
 import logging
 sys.path.append('/afs/cern.ch/cms/PPD/PdmV/tools/McM/')
 from rest import McM
+#XSDB wrapper
+from request_wrapper import RequestWrapper
 
-# McM instance
+#XSDB pycurl requester instance
+xsdb_request = RequestWrapper()
+
+
+# McM instance                                                                                                                                                                                                                                
 mcm = McM(dev=False, debug=False, cookie='cookie.txt')
 
-# Logger
+# Logger                                                                                                                                                                                                                                      
 logging.basicConfig(format='[%(asctime)s][%(levelname)s] %(message)s', level=logging.INFO)
 logger = logging.getLogger()
 
 def main():
     conn = sqlite3.connect('twiki.db')
```

Comment:
What is the purpose of `update_twiki.py` file? I see that it writes some data to `twiki.db`, which is never read and dumps the same data to `MainSamplesFall18.txt` which is then read by `update_data.py`. I would like to propose two solutions:
 * Move this file logic to `update_data.py` file, don't use `twiki.db` and `.txt` files.
 * (I would prefer to) split `update_data.py` file into two, as it is now - `update_data.py` and `update_twiki.py`. Each will be responsible only for one table in the database `data.db`.
    * `update_data.py` will fetch requests from McM and update `samples` table. That's it, it will not do anything with twiki things.
    * `update_twiki.py` will fetch certain requests from McM (>20M events), fetch information from xsdb and save them to `twiki_samples` table. It will just care about certain requests, xsdb and one table in the database.

   This way each file would have a clear purpose and responsibilities: one updates `samples`, other one `twiki_samples`. `update_data.py` already has a bunch of ifs in it's methods to have different logic for `samples` and `twiki_samples` while it should have been two separate methods. This would be a bigger task and I wouldn't want to put all this burden on @hessikaveh since `update_twiki.py` is @pgunnell 's file :thinking: 

Project is growing, so we have to think not only about gluing new "features" (_wrapping ifs in other ifs..._) to existing code, but the whole project structure and clear responsibilities of modules and their functions.